src/odemis/acq/acq_stream_settings.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+"""
+Created on 24 Jan 2024
+
+@author: Karishma Kumar
+
+Copyright © 2024-2025 Karishma Kumar, Delmic
```

Comment:
Please drop the "2025". Hopefully you're done with this feature before this date :wink: